docs/remote/containers.md

```diff
@@ -1070,42 +1107,31 @@ If you've already built the container and connected to it, be sure to run **Remo
 
 See [here for a list of active issues](https://aka.ms/vscode-remote/containers/issues) related to Containers.
 
 ### Docker limitations
 
-* First-time installs of Docker Desktop for Windows will require an additional "sharing" step to give your container access to local source code. However, this step may not work with certain AAD (email-based) identities. See [Docker Desktop for Windows tips](/docs/remote/troubleshooting.md#docker-desktop-for-windows-tips) and [Enabling file sharing in Docker Desktop](/docs/remote/troubleshooting.md#enabling-file-sharing-in-docker-desktop) for details and workarounds.
+* First-time installs of Docker Desktop for Windows when not using the WSL 2 engine will require an additional "sharing" step to give your container access to local source code. However, this step may not work with certain AAD (email-based) identities. See [Docker Desktop for Windows tips](/docs/remote/troubleshooting.md#docker-desktop-for-windows-tips) and [Enabling file sharing in Docker Desktop](/docs/remote/troubleshooting.md#enabling-file-sharing-in-docker-desktop) for details and workarounds.
 * You may see errors if you sign in to Docker with your email address instead of your Docker ID. This is a known issue and can be resolved by signing in with your Docker ID instead. See Docker [issue #935](https://github.com/docker/hub-feedback/issues/935#issuecomment-300361781) for details.
 * If you see high CPU spikes for `com.docker.hyperkit` on macOS, this may be due to a [known issue with Docker for Mac](https://github.com/docker/for-mac/issues/1759). See the Docker issue for details.
-* If you see either of these messages building a Dockerfile, you may be hitting a known Docker issue with Debian 8 (Jessie):
-
-    ```text
-    W: Failed to fetch http://deb.debian.org/debian/dists/jessie-updates/InRelease
-    E: Some index files failed to download. They have been ignored, or old ones used instead
-    ```
-
-    See [Resolving Dockerfile build failures](/docs/remote/troubleshooting.md#resolving-dockerfile-build-failures-for-images-using-debian-8) for a workaround.
 
 See the Docker troubleshooting guide for [Windows](https://docs.docker.com/docker-for-windows/troubleshoot) or [Mac](https://docs.docker.com/docker-for-mac/troubleshoot), consult [Docker Support Resources](https://success.docker.com/article/best-support-resources) for more information.
 
 ### Docker Extension limitations
 
-By default, the Docker extension will run remotely. While this is a sensible default for SSH and WSL 2, it means the extension will not show local containers when VS Code is connected to a container.
+By default, the Docker extension will run remotely. While this is a sensible default for SSH, it means the extension will not show local containers when VS Code is connected to a container.
```

Comment:
It runs locally for me. @egamma Was there a change that allows it to run locally when not installed remotely?

Reply:
The Docker extension has the UI kind set to `["workspace", "ui"]` see https://github.com/microsoft/vscode-docker/blob/24b2554622fb9f70ee414b18c2e6583a51ad357b/package.json#L10.

This means if you have installed the Docker extension locally then it will run locally.
@sandy081 fixed this issue https://github.com/microsoft/vscode/issues/83675 and this some changes for May that allows you to install a `["workspace", "ui"]` extension that you have installed locally remotely.

Bottom line this paragraph is not correct. @sandy081 what is the easiest way to install the Docker extension remotely in the April version.

Reply:
> what is the easiest way to install the Docker extension remotely in the April version.

Unfortunately there is no easiest way to install Docker extension remotely if it is already installed locally other than uninstall and install. This will remove the Docker extension locally which is unwanted. I would suggest to mention the issue in the Documentation and say that this is fixed in the next release by providing an action in the extensions view to install the extension in remote.

Reply:
Yeah at one point it installed remotely by default (type `workspace`) - this paragraph is actually old.  The same thing is in the other extensions reference guides.

I didn't know we got in a position where you couldn't click and say "install remotely" anymore.